en/docs/reference/connectors/file-connector/3.x/file-connector-3.x-example.md

```diff
@@ -54,8 +54,8 @@ Follow these steps to set up the Integration Project and the Connector Exporter
 
 9. Create the next API resource, which is `/read`. From this we are going to read the file content from a user specified location. 
 
-10. As described in step 3, drag and drop another API resource to the design view. Use the URL template as `/read`. The method will be POST. 
-    <img src="{{base_path}}/assets/img/integrate/connectors/filecon-3.x/apiResource.png" title="Adding an API resource" width="800" alt="Adding an API resource"/>
+10. As described in step 3, drag and drop another API resource to the design view. Use the URL template as `/read`. The method will be POST. <br>
+    <img src="https://apim.docs.wso2.com/en/4.1.0/assets/img/integrate/connectors/apiresource.jpg" title="Adding an API resource" width="800" alt="Adding an API resource"/>
```

Comment:
Instead of pointing to 4.1.0 docs here let's add the image to 4.2.0 space and link accordingly.